org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/thingTranslator.js

```diff
@@ -34,8 +34,8 @@ function providerSchemaToPlatformSchema(providerSchema, expand) {
             controlId: controlId
         },
         pi: providerSchema['id'],
-        mnmn: providerSchema['manufacturer'],
-        mnmo: providerSchema['model'],
+        mnmn: providerSchema['manufacturer'] === null ? "Unkown" : providerSchema['manufacturer'],
```

Comment:
Changed to return 'Undefined'.

---
In reply to: [92070701](https://github.com/openT2T/translators/pull/134#discussion_r92070701) [](ancestors = 92070701)